docs/dom-testing-library/api-queries.md

```diff
@@ -677,8 +678,12 @@ assertions about the `Open dialog`-button you would need to use
 The default value for `hidden` can
 [be configured](api-configuration#configuration).
 
-Certain roles can have a selected state. You can filter the returned elements by
-their selected state by setting `selected: true` or `selected: false`.
+#### Querying by ARIA states
+
+##### `aria-selected`
```

Comment:
Since not all `selected` elements are `aria-selected` I would omit it from the heading and just name these after their name in the options bag. Applies to checked and pressed as well.